clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkThread.java

```diff
@@ -139,6 +155,9 @@ void runOnce() {
         processApplicationEvents();
 
         final long currentTimeMs = time.milliseconds();
+        final long timeSinceLastPollMs = lastPollTimeMs != 0L ? currentTimeMs - lastPollTimeMs : currentTimeMs;
```

Comment:
In the first invocation of `runOnce()`, `timeSinceLastPollMs` will be something like `1728954137284`. Is that correct?